systeroid-tui/src/main.rs

```diff
@@ -11,29 +11,37 @@ use termion::screen::IntoAlternateScreen;
 
 fn main() -> Result<()> {
     if let Some(args) = Args::parse(env::args().collect()) {
-        let output = io::stderr();
-        let output = output.into_raw_mode()?;
-        let output = MouseTerminal::from(output);
-        let output = output.into_alternate_screen()?;
-        let backend = TermionBackend::new(output);
+        /* create a raw terminal, and immediately exit raw mode to set correct prev_ios value*/
+        let raw_output = io::stdout().into_raw_mode()?;
+        raw_output.suspend_raw_mode()?;
+
         let panic_hook = panic::take_hook();
-        panic::set_hook(Box::new(move |panic| {
+
+        panic::set_hook(Box::new (move |panic| {
             let panic_cleanup = || -> Result<()> {
-                let mut output = io::stderr();
+                let mut output = io::stdout();
```

Comment:
Should we use stderr here? The TUI runs on stderr as well.

Reply:
you mean like this?

```rust
let mut output = io::stderr();
```

Reply:
yeah, I'm not sure the usage of stdout was intentional but I prefer we go with stderr for the sake of consistency.

Reply:
oh, I understood. I'll commit with these changes.